src/ml_gym/frontend/dashboard/src/features/experimentConfig/experimentConfigSlice.tsx

```diff
@@ -0,0 +1,42 @@
+import { createSlice } from '@reduxjs/toolkit'
+import { ExperimentConfigMessageCollectionType, ExperimentConfigType} from '../../app/datatypes'
+import type { RootState } from '../../app/store';
+
+const initialState = {
+    messages: [],
+    experiment_id_to_message_index: {}
```

Comment:
Codacy has a fix for the issue: [Replace `····experiment_id_to_message_index:·{}` with `··experiment_id_to_message_index:·{},`](https://app.codacy.com/gh/mlgym/mlgym/pullRequest?prid=10020887)
```suggestion
  experiment_id_to_message_index: {},
```